src/js/components/projects/detail.tsx

```diff
@@ -100,7 +112,8 @@ const ProjectDetail = (props: RouteComponentProps) => {
           label={i18n.t('Submit Project')}
           className="slds-size_full slds-m-bottom_x-large"
           variant="outline-brand"
-          disabled
+          onClick={openSubmitModal}
+          disabled={!readyToSubmit || currentlySubmitting}
```

Comment:
Only display the button at all if `readyToSubmit`.

```suggestion
          disabled={currentlySubmitting}
```